lib/tasks/deployment/20231005170246_import_2018_onet_codes.rake

```diff
@@ -0,0 +1,20 @@
+namespace :after_party do
+  desc "Deployment task: import_2018_onet_codes"
+  task import_2018_onet_codes: :environment do
```

Comment:
How long does this task take? Is there any risk of timeout?

Reply:
Good question -- it's actually really fast, there shouldn't be any timeout issues.